src/components/home/index.module.scss

```diff
@@ -0,0 +1,13 @@
+.homeWrapper {
+  margin-top: 4.063rem;
```

Comment:
any exact reason for this kind of value? may be some round figure will looks better.

Reply:
Inside this todo app base size is 16px. While converting the space from px to rem generated this fraction points. However I removed this fractions. Thanks @iud-wtag vai